module5/lessons/Week3/Authentication.md

```diff
@@ -0,0 +1,289 @@
+---
+layout: page
+title: Authentication
+---
+
+## Learning Goals
+* See an Example of hand-rolled Auth
+* Implement the Asp.Net Core Identity Framework
+
+## An Example of Authentication and Authorization
+
+Auth (the combo of Authentication and Authorization) is an integral part of most modern web applications; and you rarely see one without the other.  Because this is such a common feature, there are **many** libraries that exist to take on most of the implementation of auth for us (yay!).  But. These libraries are large, and offer _a lot_ of functionality.  Before we dive into an Auth library, lets take a look at an implementation of auth done without the use of any special libraries.  
+
+This will help us better understand what is happening under the hood of auth libraries that you will work with _in the wild_.
+
+<section class='call-to-action' markdown='1'>
+
+Follow the instructions in the Readme of [this starter repo](https://github.com/turingschool-examples/MessageBoard)
```

Comment:
I really like that you recommend steps for them to move through when exploring the new application!